README.md

```diff
@@ -1,2 +1,27 @@
 # pydefenders
-Home repo for documentation and pointers elsewhere
+
+PyDefenders is an open source community to connect security defenders (aka blue teams) to working with Python and Jupyter tools and libraries. The use of Python is widespread among defender teams. The use of Jupyter notebooks for investigation and hunting is growing in popularity. However, finding tools and examples of code and notebooks that support defender activities is not easy.
+
+## What do we want to do?
+
+- Provide a central location where we can accumulate code, knowledge and links to other resources for defenders.
+- Members of the community are free to create and maintain repos under the pydefenders organization
+
+## Examples of Tools and Resource that we want to build and encourage
+
+- Parsing and processing for common logs and data
+- Visualizations - timelines, process trees, anomaly graphs, etc.
+- Jupyter widgets that help make authoring and reading notebooks simpler
+
+## Founding principles
+- Be agnostic to platforms and products - although we welcome contributions of tools and code developed for a specific product (e.g. ArcSight, Splunk, Azure Sentinel, etc.) the community itself is not tied and does not favor andy specific commercial or open source solutions.
+- Link rather than re-write - wherever possible, we want to point people to existing work rather than re-implement it.
+
+## Repo Organization
+This repo is intended to be a place for documentation and to store and organize links to related repos and sites. Code and notebook samples can also be submitted here but Python packages should have dedicated repos. 
+
+### Main documents
+
+- Python tools - links to python packages
+- Blogs and docs - useful documents and articles
```

Comment:
maybe something like:
```suggestion
- Blogs and docs - useful documents, articles, videos, or presentations
```
Obviously I'm biased on that one 😄  but I think there were at least a couple of talks at PyCon this year that would warrant linking.